stylix/testbed.nix

```
@@ -106,7 +106,9 @@ let
   makeTestbed =
     testbed: stylix:
     let
-      name = "testbed-${testbed.name}-${stylix.polarity}";
+      name =
+        "testbed-${testbed.name}-${stylix.polarity}"
+        + (if stylix.image == null then "-imageless" else "");
```

Consider doing the following:

```suggestion
      name =
        "testbed-${testbed.name}-${stylix.polarity}${
            lib.optionalString (stylix.image == null) "-imageless"
        }";
```

Make sure that the formatter is happy.